web/pages/index.js

```diff
@@ -0,0 +1,22 @@
+import FacebookLogin from 'react-facebook-login';
```

Comment:
Authn [freaks me out](https://twitter.com/acdlite/status/1034595817498148864). This 3rd party lib might be secure, but it MAY be safer to use the official non-react one:
https://developers.facebook.com/docs/facebook-login/web/login-button

BTW, WTH doesn't FB have a React for *Web* SDK. They only have React for mobile??
https://developers.facebook.com/docs/apis-and-sdks#official-sdks